File: 백준/Bronze/27959. 초코바/초코바.java
import java.util.Scanner;

public class Main
{
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int coins = scanner.nextInt();
		int mx = scanner.nextInt();
		
		if (coins * 100 >= mx) {
		    System.out.println("Yes");
		}
		else{
		    System.out.println("No");
		}
	}
}

File: 백준/Bronze/27959. 초코바/초코바.cc
#include <iostream>
using namespace std;

int main()
{
    int coins, mx;
    
    cin >> coins >> mx;
    
    if (coins * 100 >= mx) {
        cout << "Yes" << endl;
    }
    else{
        cout << "No" << endl;
    } 
    
    return 0;
}